src/actions/rotate/Rotate.ts

```diff
@@ -17,31 +17,31 @@ class Rotate extends Action implements IAction {
   /**
    * @return {Rotate}
    */
-  horizontalFlip(): Rotate {
+  horizontalFlip(): this {
     return this.addValue(HORIZONTAL_FLIP);
   }
 
   /**
    * @return {Rotate}
    */
-  verticalFlip(): Rotate {
+  verticalFlip(): this {
     return this.addValue(VERTICAL_FLIP);
   }
 
   /**
    * @return {Rotate}
    */
-  angle(value: number): Rotate {
+  angle(value: number): this {
     return this.addValue(value);
   }
 
-  private addValue(value?: number | string): Rotate {
+  private addValue(value?: number | string): this {
     if (value != null) {
       this.params.get(PARAM_KEY).addValue(value);
     }
     return this;
   }
 }
 
-export {angle, horizontalFlip, verticalFlip};
-export default Rotate;
+export {angle, horizontalFlip, verticalFlip, RotateAction};
+export default {angle, horizontalFlip, verticalFlip, RotateAction};
```

Comment:
Why do we need to export default anything but RotateAction?

Reply:
This will allow developers to use `import Rotate from 'actions/rotate' 

And then use it as a namespace
`Rotate.angle()` or `Rotate.verticalFlip()` 

The named exports are then used for those who want an even smaller bundle size 
`import {angle} from 'actions/rotate'`

Reply:
But this will be enough no?
export {angle, horizontalFlip, verticalFlip};
export default RotateAction;

Reply:
RotateAction is a class, and it has no static members (RotateAction.angle), so it wouldn't be exactly the same